pyteal/ast/methodsig.py

```diff
@@ -3,7 +3,6 @@
 
 from pyteal.types import TealType
 
-from ..types import TealType
```

Comment:
I'm not sure what happened here, there's both `from pyteal.types` and `from ..types`. The former should be removed, not this one

Reply:
👍